pyeudiw/tests/trust/test_TrustEvaluationHelper.py

```diff
@@ -0,0 +1,60 @@
+import pytest
```

Comment:
please, the name of this file must be lowercased and snake typed

this seems capitalized and camel typed

Reply:
Resolved [850d432](https://github.com/italia/eudi-wallet-it-python/pull/188/commits/850d432f9239edfa89a5ab2c2f9d0ea4547a50fc)